test/direct_uploads_controller_test.rb

```diff
@@ -27,7 +27,7 @@ class ActiveStorage::DirectUploadsControllerTest < ActionController::TestCase
 
       details = JSON.parse(@response.body)
 
-      assert_match /rails-activestorage\.s3.amazonaws\.com/, details["url"]
+      assert_match /#{SERVICE_CONFIGURATIONS[:s3][:bucket]}\.s3.amazonaws\.com/, details["url"]
```

Comment:
@dixpac thank you for taking a look at this.

I see what you mean, AWS returns a URL containing the region in some instances and not others. I'll work on modifying the regex so that those situations also pass.